app/database/models/ms_schema/tasks_list.py

```diff
@@ -0,0 +1,210 @@
+from enum import unique, Enum
+from datetime import date
+from app.database.db_types.JsonCustomType import JsonCustomType
+from app.database.sqlalchemy_extension import db
+
+
+class TasksListModel(db.Model):
+    """Model representation of a list of tasks.
+    
+    Attributes:
+        id: Id of the list of tasks.
+        tasks: A list of tasks, using JSON format.
+        next_task_id: Id of the next task added to the list of tasks.
+    """
+
+    __tablename__ = "tasks_list"
+    __table_args__ = {"schema": "public", "extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+    tasks = db.Column(JsonCustomType)
+    next_task_id = db.Column(db.Integer)
+
+    def __init__(self, tasks: "TasksListModel" = None):
+        """Initializes tasks.
+
+        Args:
+            tasks: A list of tasks.
+        
+        Raises:
+            A Value Error if the task is not initialized.
+        """
+
+        if tasks is None:
+            self.tasks = []
+            self.next_task_id = 1
+        else:
+            if isinstance(tasks, list):
+                self.tasks = []
+                self.next_task_id = len(tasks) + 1
+            else:
+                raise ValueError(TypeError)
+
+    def add_task(
+        self, description: str, created_at: date, is_done=False, completed_at=None
+    ) -> None:
+        """Adds a task to the list of tasks.
+        
+        Args:
+            description: A description of the task added.
+            created_at: Date on which the task is created.
+            is_done: Boolean specifying completion of the task.
+            completed_at: Date on which task is completed.
+        """
+
+        task = {
+            TasksFields.ID.value: self.next_task_id,
+            TasksFields.DESCRIPTION.value: description,
+            TasksFields.IS_DONE.value: is_done,
+            TasksFields.CREATED_AT.value: created_at,
+            TasksFields.COMPLETED_AT.value: completed_at,
+        }
+        self.next_task_id += 1
+        self.tasks = self.tasks + [task]
+
+    def delete_task(self, task_id: int) -> None:
+        """Deletes a task from the list of tasks.
+
+        Args:
+            task_id: Id of the task to be deleted.
+        """
+
+        new_list = list(
```

Comment:
use list comprehension as discussed in the meetings. Always use literal syntax [], {} instead of list() and dict()

Reply:
again, copy pasted directly from mentorship code. will do the change now